packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -333,6 +358,12 @@ export default class Jobs {
         },
         true
       )
+      type === 'backup' &&
+        app.emit('backup:postCall', {
+          ...data,
+          error: serializeError(error),
+          timestamp: Date.now(),
+        })
```

Comment:
Please document/illustrate the payloads on the PR description (pre, success and failure).

Reply:
Please document the payloads with correct cases and types.

I've edited the PR description with a proposal using TypeScript to document them.

Reply:
I wrote `VmToDefined` because I don't know what it is, it should be a bit clearer :-p